### PR TITLE
fix: cosmetic webUI improvements

### PR DESCRIPTION
**Pull request by pswid**
mainly in custom board profile and analog sensors settings. Now it looks more consistent with other pages.

### CODE REVIEW

**Comment by proddy** [comment]
I'll take a look when I'm back (traveling this week). One thing to keep in mind is that it needs to render and be usable on both a mobile phone in landscape and portrait.

File: interface/src/framework/mqtt/MqttSettingsForm.tsx [hunk 1]
```diff
@@ -219,15 +219,15 @@ const MqttSettingsForm: FC = () => {
...
+              <Grid item xs={2}>
                 <ValidatedTextField
                   name="discovery_prefix"
                   label={LL.MQTT_PUBLISH_TEXT_4()}
                   fullWidth
                   variant="outlined"
                   value={data.discovery_prefix}
                   onChange={updateFormValue}
-                  margin="normal"
+                  size="small"
```
Comment (proddy) [inline interface/src/framework/mqtt/MqttSettingsForm.tsx:230]: making it small is not consistent with the other sizes, it also renders weird and the text is cutoff in English

File: interface/src/project/SettingsApplication.tsx [hunk 2]
```diff
@@ -404,7 +405,7 @@ const SettingsApplication: FC = () => {
           />
           {data.shower_alert && (
             <>
-              <Grid item xs={4}>
```
Comment (proddy) [inline interface/src/project/SettingsApplication.tsx:407]: ok to have larger boxes if you prefer, but the spacing between the two is large and it looks funny on a desktop browser

**Review by proddy — CHANGES_REQUESTED** [review]
I reviewed this PR. I like the dialog boxes and settings improvements. Two comments
 - the shower timer & duration spacing is off
 - the MQTT HA discovery name is too small and the text is cut off

I've made individual comments

File: interface/src/framework/mqtt/MqttSettingsForm.tsx (hunk 1 above)
Comment (pswid) [reply interface/src/framework/mqtt/MqttSettingsForm.tsx:231]: IMHO, a `size="small"` style rectangle looks better than a normal (large) one when is displayed in the line with the option name. Don't you think? Also, the same style was already used in the entity language option before.

File: interface/src/project/SettingsApplication.tsx (hunk 2 above)
Comment (pswid) [reply interface/src/project/SettingsApplication.tsx:407]: my new fixes should solve the text clipping issues

**Pull request merged**